packages/opencensus-web-instrumentation-zone-peer-dep/README.md

```diff
@@ -7,15 +7,102 @@ This package generates OpenCensus Web model spans based on user interactions
 coming after initial page loads. This uses the library 
 [Zone.js](https://github.com/angular/zone.js/) to generate those spans.
 
-This package adds `Zone.js` as a peer dependency as the application 
+The library is in alpha stage and the API is subject to change.
+
+## Overview
+
+This package adds `Zone.js` as a peer dependency as your application 
 might already be dependent on it. This is the case, for example, with 
 application on `Angular` as this already imports it. 
 
-The library is in alpha stage and the API is subject to change.
+### How it works?
+
+In general, `Zone.js` is in charge of intercepting all the tasks running in an application,
+then, OC Web mokey-patches the functions `runTask`, `scheduleTask` and `cancelTask` to intercept
+and know when a task is running, being scheduled or canceled. 
+To start a new interaction, in the `runTask` monkey-patch, OC Web checks if the `task.eventName`
+corresponds to a *click* and the element that triggered the task either has a `data-ocweb-id` attibute
+or can be named with a *CSS selector*. In case those conditions are true, a new `Zone` and `rootSpan are 
+started `rootSpan` to measure the interaction.
+
+To be able to know when the interactions finishes, this is there are no more async and sync tasks related to the interaction.
```

Comment:
Would it be possible to put some sub-headers above these larger paragraphs? I think that might make it easier for someone skimming to get an idea of what's going on